utils.go

```diff
@@ -255,3 +256,24 @@ func isCSEHeader(headerKey string) bool {
 	}
 	return false
 }
+
+// sseHeaders is list of client side encryption headers
+var sseHeaders = []string{
+	"x-amz-server-side-encryption",
+	"x-amz-server-side-encryption-aws-kms-key-id",
+	"x-amz-server-side-encryption-context",
+	"x-amz-server-side-encryption-customer-algorithm",
+	"x-amz-server-side-encryption-customer-key",
+	"x-amz-server-side-encryption-customer-key-MD5",
+}
+
+// isCSEHeader returns true if header is a client side encryption header.
```

Comment:
Looks like a Typo. Needs to be

isSSEHeader returns true if header is a server side encryption header.